lib/containers/container_images.pm

```diff
@@ -265,8 +265,7 @@ sub test_systemd_install {
         ($image_id eq 'opensuse-leap' && check_version('>=' . $leap_vmin, "$image_version.$image_sp", qr/\d{2}\.\d/)) ||
         ($image_id eq 'sles' && check_version('>=' . $sle_vmin, "$image_version-SP$image_sp", qr/\d{2}-sp\d/))) {
         # Skip run case:
-        return 0 if (get_var('FLAVOR') =~ /$flavor_skip/ &&
-            $image_id eq 'sles' && check_version('<' . $bci_vmin, "$image_version-SP$image_sp", qr/\d{2}-sp\d/));
+        return 0 if ($image_id eq 'sles' && check_version('<' . $bci_vmin, "$image_version-SP$image_sp", qr/\d{2}-sp\d/));
```

Comment:
Please this check was set to solve another issue in Container Host see [18518](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18518)
May be better not to remove this flavor check

Reply:
@m-dati I run the VRs on the `Latest` and the VRs are passing.

Reply:
Yes, I added that chec to skip installing systemd for LTSS, but limited to the flavor in `flavor_skip` to resolve that Latest issue. 
Much more it works totally removing that check on flavor, because it will skip systemd install for `ALL the flavors`. 
My concern is: `is there any flavor/test-case <= 15.4` still expecting to include systemd? 
If none, than ok to totally remove flavor check.

Reply:
We need for All flavors, otherwise the BCI tests for LTSS image 15.4 on non-SLE hosts will fail.  There shouldn't be any (containers).